doc/sphinx/language/cic.rst

```diff
@@ -319,7 +319,10 @@ following rules.
    :math:`((λ x:T.~u)~t)` well-typed (where :math:`T` is a type of
    :math:`t`). This is because the value :math:`t` associated with
    :math:`x` may be used in a conversion rule
-   (see Section :ref:`Conversion-rules`).
+   (see Section :ref:`Conversion-rules`). For example
+   :g:`let T := True in (fun x : T => 42) I` is well-typed and
+   reduces to :g:`42`, while :g:`(fun T => (fun x : T => 42) I) True`
```

Comment:
Maybe the fact that `x` and `T` are not the same as in the pattern above might be a bit confusing? Might be better to use other names in the example?

Reply:
Right. Done.